armi/operators/operator.py

```diff
@@ -484,8 +483,6 @@ def _interactAll(self, interactionName, activeInterfaces, *args):
                     )
                 )
 
-            self._checkCsConsistency()
```

Comment:
Instead of getting rid of this check entirely, do we instead just want to ensure that the `cs` on interface is consistent among them all?

Reply:
I can look at it again.

But it really looks like the only thing this function does is check that your CS object matches the master/global CS object.  So, I _expect_ that this function will suddenly do nothing once we remove the master/global CS object.  I'll look.

Reply:
Okay, unless you have another idea, this method simply compares `o.cs` to the global CS to see if they match.

Now that there is only one CS object possible, and not two, I don't think we need this method any more.